tutorials/demo.rst

```diff
@@ -8,6 +8,8 @@ OpenWISP Demo
    :backlinks: none
    :depth: 3
 
+.. _accessing_the_demo_system:
+
```

Comment:
Please add a link to the new tutorial at the bottom of the "next" section of this page.